src/GrumPHP/EventListener/TaskEventListener.php

```diff
@@ -37,83 +37,106 @@ class TaskEventListener
-            $key_prefix . 'PACKAGE_TYPE' => $package_path . $info['filename'] . '-' . $type_suffix . '.' . $info['extension'],
-            $key_prefix . 'PACKAGE_GLOBAL' => $package_path . $info['filename'] . '.' . $info['extension'],
+            $keyPrefix . 'LOCAL' => sprintf(
+                '%s.local.%s',
+                $info['filename'],
+                $info['extension']
+            ),
+            $keyPrefix . 'PROJECT' => sprintf(
+                '%s.%s',
+                $info['filename'],
+                $info['extension']
+            ),
+            $keyPrefix . 'PACKAGE_TYPE' => sprintf(
+                '%s%s-%s.%s',
+                $packagePath,
+                $info['filename'],
+                $typeSuffix,
+                $info['extension']
+            ),
+            $keyPrefix . 'PACKAGE_GLOBAL' => sprintf(
+                '%s%s.%s',
+                $packagePath,
+                $info['filename'],
+                $info['extension']
+            ),
         ];
 
         // Search for the candidates and merge or copy them.
-        $fs = new Filesystem();
-        $data_merged = NULL;
+        $filesystem = new Filesystem();
+        $dataMerged = null;
 
         foreach ($candidates as $env_var => $file) {
             // Ignore if configured to skip or if the file is missing.
-            if (!empty($_SERVER[$env_var]) || !$fs->exists($file)) {
+            if (!empty($_SERVER[$env_var]) || !$filesystem->exists($file)) {
                 continue;
             }
 
             // Read and parse the configuration file.
             $data = $this->readTaskConfigFile($info['type'], $file);
 
-            if ($data === FALSE) {
+            if ($data === false) {
                 // Just copy it if not readable.
-                $fs->copy($file, $info['grumphp']);
+                $filesystem->copy($file, $info['grumphp']);
                 return;
             }
 
             // Merge the data.
-            if ($data_merged === NULL) {
-                $data_merged = $data;
-            }
-            elseif ($data) {
-                $data_merged = array_merge_recursive($data, $data_merged);
+            if ($dataMerged === null) {
+                $dataMerged = $data;
+            } elseif ($data) {
+                $dataMerged = array_merge_recursive($data, $dataMerged);
             }
         }
 
         // Save the configuration file.
-        $this->writeTaskConfigFile($info['type'], $info['grumphp'], $data_merged);
+        $this->writeTaskConfigFile($info['type'], $info['grumphp'], $dataMerged);
     }
 
     /**
      * Remove the task configuration file.
      *
-     * @param TaskEvent $event The GrumPHP task event.
+     * @param \GrumPHP\Event\TaskEvent $event
+     *   The GrumPHP task event.
      */
-    public function removeTaskConfig(TaskEvent $event)
+    public function removeTaskConfig(TaskEvent $event): void
```

Comment:
Als je die niet meer gebruikt mag die weg zeker?

Reply:
Is removed.